common/autoware_fake_test_node/include/fake_test_node/fake_test_node.hpp

```diff
@@ -20,7 +20,7 @@
 #ifndef FAKE_TEST_NODE__FAKE_TEST_NODE_HPP_
 #define FAKE_TEST_NODE__FAKE_TEST_NODE_HPP_
 
-#include <fake_test_node/visibility_control.hpp>
+#include <autoware/fake_test_node/visibility_control.hpp>
```

Comment:
This include fails because there is no `autoware` subfolder.

Reply:
Thanks for spotting this. Fixed in the latest changes.